src/main/java/jp/co/soramitsu/sora/json/Hashifier.java

```diff
@@ -0,0 +1,60 @@
+package jp.co.soramitsu.sora.json;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.security.MessageDigest;
+import java.util.ArrayList;
+import java.util.List;
+import jp.co.soramitsu.jackson.OneCodeMapper;
+import lombok.NonNull;
+import lombok.Setter;
+
+public class Hashifier {
+
+  @Setter
+  private MessageDigest digest;
+
+  @Setter
+  private ObjectMapper mapper = new ObjectMapper();
+
+  private ObjectMapper onecoder = new OneCodeMapper();
+
+  public Hashifier(@NonNull MessageDigest digest) {
```

Comment:
As I understand, this class allows `digest` to be changed after object was created using setter. Because of that, I don't see any point to enforce passing `digest` to constructor, as client can do that after object was created. 

I suggest removing `NonNull` and adding default constructor

Reply:
Object without message digest is not valid, since MessageDigest is an interface. Should I remove setter instead?